kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/cache/Reflector.java

```diff
@@ -111,6 +111,15 @@ private void reListAndSync() {
     }
   }
 
+  private void stopWatcher() {
```

Comment:
Let's reuse the existing stop method instead of introducing a new method.

Reply:
ok, so, I will update my PR to call the `stop` method instead.

Reply:
@akram sorry, I didn't go back and update all of my comments - we don't want to call stop as written.